From 33c15e3ac353a5594efb9815f186d96b0f821f3f Mon Sep 17 00:00:00 2001
From: Adhemerval Zanella <azanella@linux.vnet.ibm.com>
Date: Mon, 4 Mar 2013 11:37:51 -0300
Subject: [PATCH 16/42] BZ #15055: Use __ieee754_sqrl in acoshl for
 lbdl-128ibm (backported from commit
 e0b780ad5b94209bf99bf498314bc5c160dc2a15)

---
 sysdeps/ieee754/ldbl-128ibm/e_acoshl.c | 2 +-
 3 files changed, 8 insertions(+), 2 deletions(-)

diff --git glibc-2.17-c758a686/sysdeps/ieee754/ldbl-128ibm/e_acoshl.c glibc-2.17-c758a686/sysdeps/ieee754/ldbl-128ibm/e_acoshl.c
index 117bd0f..abc78a3 100644
--- glibc-2.17-c758a686/sysdeps/ieee754/ldbl-128ibm/e_acoshl.c
+++ glibc-2.17-c758a686/sysdeps/ieee754/ldbl-128ibm/e_acoshl.c
@@ -52,7 +52,7 @@ __ieee754_acoshl(long double x)
 	    return __ieee754_logl(2.0*x-one/(x+__ieee754_sqrtl(t-one)));
 	} else {			/* 1<x<2 */
 	    t = x-one;
-	    return __log1p(t+__sqrtl(2.0*t+t*t));
+	    return __log1p(t+__ieee754_sqrtl(2.0*t+t*t));
 	}
 }
 strong_alias (__ieee754_acoshl, __acoshl_finite)
-- 
1.7.11.7

